style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+  margin: auto;
+  background-color: #344563;
+  font-family: "Poppins", sans-serif;
+}
+
+.header {
+  width: 375px;
+  height: 48px;
+  top: 44px;
+  position: absolute;
+  background-image: url("Media/header.png.jpg");
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  top: 10px;
+  left: 24px;
+  font-weight: 700px;
+  font-size: 18px;
+  line-height: 20px;
+  align-items: center;
+}
+
+.link {
+  text-decoration: none;
+}
+
+.header .logo {
+  width: 100px;
+  height: 40px;
+  top: 4px;
+  left: 12px;
+  padding: 10px 12px;
+}
+
+.menubar {
+  width: 16px;
+  height: 0;
+  top: 6px;
+  left: 3px;
+}
+
+.header .menu {
+  width: 34px;
+  height: 44px;
+  top: 19px;
+  right: 24px;
+  flex-direction: row;
+  align-items: flex-start;
+  margin-left: -35px;
+  padding: 8px;
+}
+
+.headline {
+  width: 375px;
+  height: 604px;
+  top: 92px;
+  padding: 114px 24px;
+  gap: 12px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: absolute;
+  background-size: cover;
+  background-image: url("Media/background.jpg");
+}
+
+.headline .me {
+  width: 327px;
+  height: 150px;
+  line-height: 52px;
+  font-size: 32px;
+  font-style: normal;
+  flex: none;
+  flex-grow: 0;
+  align-items: center;
+  display: flex;
+  font-weight: 700px;
+}
+
+.headline .dec {
+  width: 327px;
+  height: 146px;
+  top: 276px;
+  left: 24px;
+  font-weight: 400px;
+  font-size: 16px;
+  line-height: 24px;
+  flex-grow: 0;
+  order: 1;
+}
```

Comment:
- [ ] Please adhere to the design as much as possible, as I can see you are not using the right colour. Please keep in mind that as a developer it is very important to have a *detail-oriented* mindset because it will help you to make a better developer who always takes care of small details to make the design looks professional and consistent. 

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-01 16-09-20](https://user-images.githubusercontent.com/66476329/235441793-6628a2f1-cab4-4e29-8082-cf761109336c.png)

</td>
<td>

![Screenshot from 2023-05-01 14-54-23](https://user-images.githubusercontent.com/66476329/235441734-9e7f6775-79ff-4cdf-b157-06257c0821fe.png)

</td>
</tr>
</table>